backend/inspirehep/mailing/api/jobs.py

```diff
@@ -38,3 +41,36 @@ def send_jobs_weekly_campaign(html_content, test_emails=None):
 def subscribe_to_jobs_weekly_list(email, first_name, last_name):
     list_id = current_app.config["MAILCHIMP_JOBS_WEEKLY_LIST_ID"]
     return mailchimp_subscribe_user_to_list(list_id, email, first_name, last_name)
+
+
+def get_jobs_deadline_reminder_html_content(job):
+    host = current_app.config['SERVER_NAME']
+    return render_template("mailing/jobs/deadline_passed/base.html", job=job, host=host)
+
+
+def send_job_deadline_reminder(job):
+    """Send an email for the given expired job.
+
+    Args:
+        job (dict): a job record.
+
+    Return:
+        None
+    """
+    receiver = job.get("acquisition_source", {}).get("email")
```

Comment:
* `recipient` (everywhere)
* I think you should use the email in the user account, which in principle might be more up-to-date.